### PR TITLE
remove warnings in CumulativeSum 

### PR DESCRIPTION
**Pull request by adamkozuch**
Issue #1211

### CODE REVIEW

File: scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ import java.io.OutputStream
 import java.io.Serializable
```
Comment (johnynek) [inline scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala]: let's not add this change.

File: scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala [hunk 2]
```diff
@@ -37,7 +37,12 @@ object CumulativeSum {
     val pipe: TypedPipe[(K, (U, V))]) {
     /** Takes a sortable field and a monoid and returns the cumulative sum of that monoid **/
     def cumulativeSum(
-      implicit sg: Semigroup[V], ordU: Ordering[U], ordK: Ordering[K]): SortedGrouped[K, (U, V)] = {
+      implicit sg: Semigroup[V],
+      ordU: Ordering[U],
+      ordK: Ordering[K],
+      tagK: ClassTag[K],
```
Comment (johnynek) [inline scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala]: let's not use the ClassTag. That will complicate the use of the code.

File: scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala [hunk 3]
```diff
@@ -60,7 +65,10 @@ object CumulativeSum {
...
       sg: Semigroup[V],
       ordU: Ordering[U],
-      ordK: Ordering[K]): TypedPipe[(K, (U, V))] = {
+      ordK: Ordering[K],
+      tagK: ClassTag[K],
+      tagV: ClassTag[V],
+      tagU: ClassTag[U]): TypedPipe[(K, (U, V))] = {
 
       val sumPerS = pipe
         .map { case (k, (u: U, v: V)) => (k, partition(u)) -> v }
```
Comment (johnynek) [inline scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala]: actually, I think it is as easy as removing the `U` and `V` and not adding the ClassTags.

**Comment by adamkozuch** [comment]
Thanks for advice. Now it passed. Is that all?

**Comment by johnynek** [comment]
Thank you sir! Very helpful.

**Pull request merged**